ContactsApp/ContactsApp/ContactsControllers/ContactViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ContactViewController.swift
+//  ContactsApp
+//
+//  Created by Apple Macbook Pro 13 on 13.08.22.
+//
+
+import UIKit
+// MARK: Protocol
+protocol UpdateContact {
+    func updateContact(model: Person)
+}
+
+class ContactViewController: UIViewController {
+
+    // MARK: IBOutlets
+    @IBOutlet private weak var phoneNumber: UILabel!
+    @IBOutlet private weak var emailAddress: UILabel!
```

Comment:
Add type of UI component as suffix to all IBOutlets:
`emailAddress -> emailLabel`